value/binary.go

```diff
@@ -1172,6 +1172,32 @@ func init() {
 		},
 
 		{
+			name:      "flip",
+			whichType: atLeastVectorType,
+			fn: [numType]binaryFn{
+				vectorType: func(c Context, u, v Value) Value {
+					countVec := u.(Vector)
+					count, ok := countVec[0].(Int)
```

Comment:
You have to check the length before you index. Unlikely to cause problems but it's possible to have a zero-length vector.

It's possible other places make this mistake, but don't worry about them now. I'll file an issue about it.